game-headed/src/main/java/org/triplea/game/client/ui/javafx/util/FxmlManager.java

```diff
@@ -19,7 +19,9 @@
 
   GAME_SELECTION_CONTROLS("/org/triplea/game/client/ui/javafx/fxml/GameSelectionControls.fxml"),
 
-  MAIN_MENU_CONTROLS("/org/triplea/game/client/ui/javafx/fxml/MainMenuControls.fxml");
+  MAIN_MENU_CONTROLS("/org/triplea/game/client/ui/javafx/fxml/MainMenuControls.fxml"),
+
+  MAP_SELECTION("/org/triplea/game/client/ui/javafx/fxml/MapSelection.fxml");
```

Comment:
nit, to avoid unnecessary diffs in future, ending enum with comma and putting semi on new line will keep changes a bit easier to grok. EG:
```
MAP_SELECTION("/org/triplea/game/client/ui/javafx/fxml/MapSelection.fxml"),
;
```